pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -46,8 +47,17 @@ var _ Interface = new(cadvisorClient)
 // The amount of time for which to keep stats in memory.
 const statsCacheDuration = 2 * time.Minute
 const maxHousekeepingInterval = 15 * time.Second
+const defaultHousekeepingInterval = 10 * time.Second
 const allowDynamicHousekeeping = true
 
+func init() {
+	// Override the default cAdvisor housekeeping interval.
+	if f := flag.Lookup("housekeeping_interval"); f != nil {
```

Comment:
This probably needs a bigger discussion, as it would not be a backwards-compatible change. We'd probably need to clone all the flags into kubelet and mark them as depricated for a long time.